packages/winston-log-enricher/tests/unit/auto.tap.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const utils = require('@newrelic/test-utilities')
+const formatFactory = require('../../lib/createFormatter.js')
+const API = require('newrelic/api')
+
+utils(tap)
+
+tap.test('auto-instrumentation of winston', { bail: true }, (t) => {
```

Comment:
the titles feel weird now that they are unit tests but that's fine. Also it's a little confusing that the unit test file is called auto.tap.js, that doesn't really mean anything to me in the context of unit tests but I don't want to keep going back and forth here. we can adjust later